src/languages/stan.js

```diff
@@ -484,14 +325,14 @@ export default function(hljs) {
 
   const INCLUDE = {
     className: 'meta',
-    begin: /^#include\b/,
+    begin: /#\s*[a-z]+\b/,
     end: /$/,
     relevance: 0, // relevance comes from keywords
-    keywords: "include",
+    keywords: '#include',
```

Comment:
Pretty sure this won't work without using `$pattern` as # isn't a normal keyword character.

Reply:
It's parsing as what I want:
![Screen Shot 2021-11-30 at 1 45 47 PM](https://user-images.githubusercontent.com/23704057/144108623-0917f953-65c0-4c84-8463-4da1def3d1a8.png)

Feel free to make changes. I don't know javascript or regex.

Reply:
You only seeing the `meta` highlighting... the `keywords` isn't working there at all because the "#" is not matched by IDENT_RE, which is what Stan uses for keyword pattern matching.

Reply:
I removed keywords and just use the meta highlighting as I think it looks good. Let me know if this goes against the package design philosophy.

Reply:
It's probably ok... I'll make a final pass and clean it up later if necessary.  The relevancy may have to be tuned now, etc... but it likely looked weird before for some themes so I'm ok changing it.